test/shared_setup.py

```diff
@@ -219,3 +219,37 @@ def test_parse_target(self):
         self.assertTrue(new_target['long_description'].startswith("Description\n==========="))
 
         klass.SHARED_TARGET = orig_target
+
+    def test_parse_target_dependencies(self):
+        """Test injecting dependency_links in parse_target"""
+        package = {
+            'name': 'vsc-test',
+            'excluded_pkgs_rpm': [],
+            'version': '1.0',
+            'install_requires': [
+                'vsc-config >= 2.0.0',
+                'vsc-accountpage-clients',
+                'vsc-base > 1.0.0'
+            ],
+        }
+        setup = vsc_setup()
+        # this is needed to pass the tests on Travis: travis will clone vsc-install through
+        # https and it will be marked as non private repo, causing the dependency_links to be injected
+        # with git+https, which is not correct in this test case.
+        setup.private_repo = True
```

Comment:
@wpoely86 Then make this conditional, only set this when the `$TRAVIS` environment variable is defined?

Reply:
it's going to be True in most cases (like on my laptop or Jenkins)?